src/Extensions/hcox_soilnox_mod.F90

```diff
@@ -415,7 +415,11 @@ SUBROUTINE HCOX_SoilNOx_Run( ExtState, HcoState, RC )
     ! after the rewinding the clock.
     !---------------------------------------------------------------
 
+#if !defined ( MODEL_GEOS ) 
     IF ( FIRST .OR. HcoClock_Rewind( HcoState%Clock, .TRUE. ) ) THEN
+#else
+    IF ( .TRUE. ) THEN
```

Comment:
Same question here as earlier - could you give a brief summary on why this change is needed for GEOS?